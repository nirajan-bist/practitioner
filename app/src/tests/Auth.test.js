import { render, screen, fireEvent, waitFor } from "@testing-library/react";
import Auth from "components/auth";
import { Provider } from "react-redux";
import store from "store";

it("Render children if the user is not logged in", () => {
  render(
    <Provider store={store}>
      <Auth>Children </Auth>
    </Provider>
  );
  expect(screen.getByText("Children")).toBeInTheDocument();
});

it("Render children and match the snapshot", async () => {

 const {asFragment}= render(
    <Provider store={store}>
      <Auth >Children </Auth>
    </Provider>
  );
  expect( screen.getByText("Children")).toBeInTheDocument();
  expect(await asFragment()).toMatchSnapshot();
});
