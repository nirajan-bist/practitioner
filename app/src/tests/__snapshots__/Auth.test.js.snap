// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render children and match the snapshot 1`] = `
<DocumentFragment>
  Children 
</DocumentFragment>
`;
